core/controllers/skill_mastery.py

```diff
@@ -173,9 +172,8 @@ def get(self):
                 if skill_mastery_dict:
                     # Subtopic mastery is average of skill masteries.
                     subtopic_mastery_dict[topic.id][subtopic.id] = (
-                        python_utils.divide(
-                            sum(skill_mastery_dict.values()),
-                            len(skill_mastery_dict)))
+                        sum(skill_mastery_dict.values()) / len(
```

Comment:
Why is it `/` here, but `//` in some other places? Can we have the same symbol everywhere?